src/network/network.hpp

```diff
@@ -53,6 +53,14 @@ class Network {
   bool pbft_syncing();
   uint64_t syncTimeSeconds() const;
 
+  uint64_t pbftSyncingPeriod() const;
+  void syncBlockQueuePop();
+  std::shared_ptr<std::pair<SyncBlock, dev::p2p::NodeID>> processSyncBlock();
+  void syncBlockQueuePush(SyncBlock const &block);
+  void clearSyncBlockQueue();
+  size_t syncBlockQueueSize() const;
+  void handleMaliciousSyncBlock(NodeID const &id);
```

Comment:
So I went through the changes and I can see you moved `sync_queue_` from `pbft_manager` to `taraxa_capability` but I cant really see any added value in a way it is done (sure I might be wrong but this is how I understand the changes from quick review). Let me explain:

- I still think that the ultimate goal is to remove processing of sync blocks from pbft_manager and move it all somehow to tarcap. Then during syncing pbft_manager could be sort of "disabled"... 
- What we have here is that all the processing of sync blocks is still inside pbft_manager and as a result of moving `sync_queue_` into the tarcap is that you had to create 7 new methods inside network class as well as tarcap class, which is a lot of methods. 
- tarcap in fact does not really use `sync_queue_` except pushing blocks into it.  Only pbft_manager depends on it and use it internally. So if you leave the queue inside pbft_manager you can delete those 14 methods. I would keep maybe only adjusted `handleMaliciousSyncBlock()` function inside tarcap that would set malicious flag and disconnect the peer...

In summary, as we did not  remove also actual processing of sync blocks from pbft_manager, these changes seem like it complicated stuff rather then improved it ?

Reply:
I have made some additional changes after your comment. There was some leftover processing in pbft manager that I also moved to tarcap. Network and tarcap have this issue where this maybe should be a single class since it is unlikely that we would have other capabilities. Currently only 4 methods are exposed in network that deal with the queue.

Reply:
I dont see any changes that would move sync pbft blocks processing from pbft_manager to tarcap, so the comment from above still holds